documentation/docs/contributing/development-guide/05-publishing.md

```diff
@@ -25,21 +21,21 @@ Release to production is triggered when we merge in the auto-generated pull requ
 
 A reusable workflow is just like any GitHub Actions workflow with one key difference - it includes a `workflow_call` trigger.
 
-* The `release.yaml` workflow will automatically create a pull request with an auto-generated changelog everytime code is pushed to the main branch. It will also at the same time bump version with standard-version and create a tagged release that can be used to deploy to production.
-  * Missing: How to deploy the project on a live system.
+* The `release-please.yaml` workflow will automatically create a pull request with an auto-generated changelog everytime code is pushed to the main branch. it also bumps the version code (using semantic versioning, depending on the types of commits)  and create a tagged release that can be used to deploy to production.
```

Comment:
```suggestion
* The `release-please.yaml` workflow will automatically create a pull request with an auto-generated changelog everytime code is pushed to the main branch. It also bumps the version code (using semantic versioning, depending on the types of commits) and create a tagged release that can be used to deploy to production.
```

Reply:
This error is part of my correction, so I'll have to take blame for this one. 

![tenor-204806489](https://user-images.githubusercontent.com/37065184/196355758-c150ae88-2bbd-4541-af24-ef8d192e6522.gif)